protocol/message.go

```diff
@@ -219,19 +219,28 @@ func NewMonitoringProof(ap []*m.AuthenticationPath,
 // If the response contains a range of authentication paths,
 // the key is obtained from the authentication path corresponding
 // with the most recent signed tree root.
-func (msg *Response) GetKey() []byte {
+func (msg *Response) GetKey() ([]byte, error) {
+	// repeat some assertions from consistencychecks.go
```

Comment:
Yes, I was thinking it too. Please see e3c4cf418fa2dffad29c264f9bbb3dc0e020ff5c.